README.md

```diff
@@ -58,7 +58,7 @@ Tailor uses [sax](https://github.com/isaacs/sax-js) to parse the template, where
 * *timeout* - optional timeout of fragment in milliseconds (default is 3000)
 * *async* — postpones the fragment until the end of body tag
 * *public* — doesn't send the headers to fragment server
-* *fallback-src* - URL of the fallback fragment in case of timeout/error on the current fragment
+* *fallback_src* - URL of the fallback fragment in case of timeout/error on the current fragment
```

Comment:
I think that hyphen should be used for html attributes, its a de facto standard, e.g. https://developer.mozilla.org/en/docs/Web/Guide/HTML/Using_data_attributes 

Using underscores will make html look weird